src/terminal.go

```diff
@@ -1758,6 +1759,46 @@ func (t *Terminal) executeCommand(template string, forcePlus bool, background bo
 	cleanTemporaryFiles()
 }
 
+func (t *Terminal) executeChangeQuery(template string) {
+	valid, list := t.buildPlusList(template, false)
+	if !valid {
+		return
+	}
+	command := t.replacePlaceholder(template, false, string(t.input), list)
+	cmd := util.ExecCommand(command, false)
+	newQuery := []rune{}
+	t.executing.Set(true)
+	cmd.Stdin = tui.TtyIn()
+	out, _ := cmd.StdoutPipe()
+	t.tui.Pause(false)
+	cmd.Stderr = cmd.Stdout
+	reader := bufio.NewReader(out)
+	err := cmd.Start()
+	if err != nil {
+		newQuery = append(newQuery, []rune(err.Error())...)
+	} else {
+		for {
+			line, err := reader.ReadString('\n')
```

Comment:
This works windows and *nix line endings. But then you may have `\n` or `\r\n` at the end of the string.